scripts/checks/verify_supply_chain.py

```
@@ -645,6 +661,168 @@ def verify_release_asset_allowlist_policy() -> list[str]:
     return violations
 
 
+def rust_dependency_advisory_violations(
```

Ruff (0.15.12)

[warning] Too many branches (14 > 12)

(PLR0912)